chapters/VK_EXT_debug_utils.txt

```diff
@@ -44,6 +44,14 @@ include::../api/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
 include::../validity/protos/vkSetDebugUtilsObjectNameEXT.txt[]
 
+.Valid Usage
+****
+  * pname:pNameInfo\->pname:objectType must: not be ename:VK_OBJECT_TYPE_UNKNOWN
+  * pname:pNameInfo\->pname:objectHandle must: be a Vulkan handle of the type associated with
+    pname:pNameInfo\->pname:objectType as defined in
+    <<debugging-object-types, VkObjectType and Vulkan Handle Relationship>>.
```

Comment:
"in the ... table"?
\+ No period after VU

Reply:
:+1: